corgi/core/constants.py

```diff
@@ -84,7 +84,7 @@
 # So the SQL syntax will be correct in the final stored procedure code
 LATEST_FILTER_DEFINITION = (
     "get_latest_component( "
-    "model_type text, ps_ofuri text, component_type text, component_ns text, "
+    "model_type text, ps_ofuris text[], component_type text, component_ns text, "
```

Comment:
I think we can drop all the changes to constants.py in this commit right? We are now adding a new database function.

Reply:
yes of course you are right ! thx